x-pack/plugins/data_usage/server/routes/internal/usage_metrics_handler.ts

```diff
@@ -54,6 +54,7 @@ export const getUsageMetricsHandler = (
         to,
         metricTypes: formatStringParams(metricTypes) as MetricTypes[],
         dataStreams: formatStringParams(dataStreamsResponse.map((ds) => ds.name)),
+        autoOpsAPIService: autoopsApiService,
```

Comment:
Maybe it is better to add the autoOpsApiService to the appContextService and have it initialize it at plugin start.